nestkernel/conn_builder.cpp

```diff
@@ -83,15 +83,14 @@ nest::ConnBuilder::ConnBuilder( const GIDCollection& sources,
     throw UnknownSynapseType( syn_name );
   }
 
-  // if another synapse than static_synapse is defined we need to make
-  // sure that Connect can process all parameter specified
-  if ( syn_name != "static_synapse" )
-  {
-    check_synapse_params_( syn_name, syn_spec );
-  }
-
   synapse_model_ = kernel().model_manager.get_synapsedict()->lookup( syn_name );
 
+  // We need to make sure that Connect can process all synapse parameters
+  // specified.
+  ConnectorModel& syn_mod =
```

Comment:
This could be `const`, and would suggest the following renaming:
- `synapse_model_` -> `synapse_model_id_` (because it just is the id number)
- `syn_mod`-> `synapse_model`